br/tests/lightning_extend_routes/run.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Basic check for whether routing rules work
+
+set -eux
+
+backends=("local" "tidb")
+for backend in ${backends[@]}; do
+  run_sql 'DROP DATABASE IF EXISTS routes_a0;'
+  run_sql 'DROP DATABASE IF EXISTS routes_a1;'
+  run_sql 'DROP DATABASE IF EXISTS routes_b;'
+
+  run_sql 'CREATE DATABASE routes_b;'
+  run_sql 'CREATE TABLE u (x real primary key, c_source varchar(11) not null, c_schema varchar(11) not null, c_table varchar(11) not null);'
```

Comment:
will it work without create table explicitly?

Reply:
No. Users have to create table with extended columns manually.